src/utils.ts

```diff
@@ -76,15 +80,20 @@ export function createCITelemetryData(telemetryData: ProcessTelemetryDatum): CIT
 
 export async function sendData (url :string, ciTelemetryData: CITelemetryData)
 {
-    logger.debug(`Sending data (api key=${core.getInput("api_key")}) to url: ${url}`);
+    const apiKey = await getApiKey(ciTelemetryData.metaData);
+    if (apiKey == null) {
+      logger.debug(`ApiKey is not exists! Data can not be send.`);
```

Comment:
I think, this should be error level log. Also we should be sure that when we print error message here, the workflow run doesn't fail because of it.

Reply:
done